libbeat/docs/newdashboards.asciidoc

```diff
@@ -0,0 +1,149 @@
+[[new-dashboards]]
+== Developer Guide: Creating new Kibana dashboards
+
+This guide walks you through the steps for creating a new Kibana dashboards
```

Comment:
We should recommend here to use the virtual environment under `testing/environments` to create the dashboards. The reasons for this is:

* People start with an empty Kibana instance
* Everyone is using the same Kibana version (most recent one)
* No setup on developer side needed

This should lead to hopefully cleaner dashboards or outdated dashboards.